portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,25 @@
   </head>
   <body>
     <div id="content">
-      <h1>Yenessa's Portfolio</h1>
-      <p>This is my portfolio!! :D</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Ayyy</button>
-      <div id="greeting-container"></div>
-      <p>I'm currently a sophomore studying Computer Engineering at the University of Florida. I'm passionate about the societal impact of technology and the integration between hardware and software.</p>
-      <p>I also loooove cats! Here's one of my favorite cat images. </p>
-      <img src="images/catinbox.JPG"/>
+        <body style="background-color:thistle;">
+        <font face="garamond">
+            <h1>Yenessa's Portfolio</h1>
+            <a href="https://www.linkedin.com/in/yenavalos/" style="color:steelblue"><h4><i>Check out my LinkedIn page!</i></h4></a>
```

Comment:
If you want this to open in a new tab/window add `target="_blank"` inside the `<a>` tag as an attribute.

Ex:
`<a href="https://www.linkedin.com/in/yenavalos/" style="color:steelblue" target="_blank"><h4><i>Check out my LinkedIn page!</i></h4></a>`

I'd also again suggest that styling (color, italics) goes into a CSS file.

`font-style: italic;` and the color

You can just apply to the `h4` so every h4 you add picks it up.
`h4 { font-style: italic; }` 

The `<i>` formatting tag isn't really used anymore. Same with `<b>` and `<u>`.